Frontend/ui-driver/src/Components/DueDetailsList/View.purs

```diff
@@ -48,6 +50,7 @@ view push state =
        ](mapWithIndex
          (\index item ->
           let rideNumberPrefix = if item.noOfRides < 10 then "0" else ""
+              _ = spy "value of index" item.amountPaidByYatriCoins
```

Comment:
@DwivediAman-exe is this spy statement required? if not then remove the import as well.